src/exceptions/utils.ts

```diff
@@ -76,23 +62,34 @@ const DEFAULT_MAP: HTTPErrCodeExceptionMap = {
  * );
  * ```
  */
-export const throwExpected = async <T = any>(fn: () => Promise<T>, exceptionsMap: HTTPErrCodeExceptionMap = {}): Promise<T> => {
+export const throwExpected = async <T = any>(fn: () => Promise<T>, exceptionsMap: HTTPErrCodeExceptionMap = {}) => {
   try {
     const result = await fn();
 
     return result;
-  } catch (e: unknown) {
-    const response = await toPlatformErrorResponse((e as any)?.response);
-    const errCode = response?.status! ?? parseHTTPErrorCode(e) ?? FALLBACK_HTTP_ERROR_CODE;
-    const mappedExceptionFn = exceptionsMap[errCode] ?? DEFAULT_MAP[errCode];
+  } catch (error: unknown) {
+    if (axios.isAxiosError(error)) {
+      const httpStatusCode = error.response?.status ?? parseHTTPErrorCode(error);
+
+      if (!httpStatusCode) {
+        throw new LensSDKException(httpStatusCode, "Unexpected exception [Lens Platform SDK]", error);
+      }
 
-    if (mappedExceptionFn) {
-      // Throw expected exception
-      throw mappedExceptionFn(response);
-    } else {
-      // Throw strongly-typed unexpected exception
-      throw new LensSDKException(errCode, `Unexpected exception [Lens Platform SDK]: ${HTTPErrorCodes[errCode]}`, e);
+      const mappedExceptionFn = exceptionsMap[httpStatusCode];
+
+      if (mappedExceptionFn) {
+        throw mappedExceptionFn(
+          await toPlatformErrorResponse(error?.response)
+        );
+      } else if (
+        httpStatusCode === 401 || httpStatusCode === 403
+      ) {
+        throw new LensSDKException(httpStatusCode, error.response?.data?.message, error);
```

Comment:
Shouldn't be throw ForbiddenException or UnauthorizedException? Otherwise this is a breaking change.

Reply:
Also, does `error.response?.data?.message` equal `error?.body.message`? If not, there is breaking change in src/UserService.ts.

Reply:
`error.response?.data?.message equal error?.body.message` <== yes

Reply:
This should be a major version anyway.